CI/macos/03_package_obs.sh

```diff
@@ -134,11 +134,11 @@ package-obs-standalone() {
 
     if [ -z "${NOTARIZE_IMAGE}" -a -z "${NOTARIZE_BUNDLE}" ]; then
         if [ "${ARCH}" = "arm64" ]; then
-            FILE_NAME="obs-studio-${VERSION_STRING}-macOS-Apple.dmg"
+            FILE_NAME="obs-studio-macOS-${VERSION_STRING}-arm64.dmg"
         elif [ "${ARCH}" = "universal" ]; then
-            FILE_NAME="obs-studio-${VERSION_STRING}-macOS.dmg"
+            FILE_NAME="obs-studio-macOS-${VERSION_STRING}.dmg"
         else
-            FILE_NAME="obs-studio-${VERSION_STRING}-macOS-Intel.dmg"
+            FILE_NAME="obs-studio-macOS-${VERSION_STRING}-x86_64.dmg"
```

Comment:
Prefer that the OBS Studio version string is immediately after "obs-studio" to associate the two pieces of information.
```
obs-studio-${VERSION_STRING}-macos-arm64.dmg
```
```
obs-studio-${VERSION_STRING}-macos.dmg
```
```
obs-studio-${VERSION_STRING}-macos-x86_64.dmg
```

Reply:
fixed